src/client/datascience/notebook/notebookControllerManager.ts

```diff
@@ -145,8 +145,11 @@ export class NotebookControllerManager implements INotebookControllerManager, IE
             const controllers = await this.createNotebookControllers(connections);
 
             // Send telemetry related to fetching the kernel connections
-            // KERNELPUSH: undefined works for telemetry?
-            sendNotebookControllerCreateTelemetry(undefined, controllers, stopWatch);
```

Comment:
This was incorrect, we were sending `undefined`, hence the resource was not a notebook, and the mapping of kernel information for the notebooks was not showing up in telemetry.